screens/OnboardingScreen.js

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { View } from 'react-native';
+import { t } from 'react-native-tailwindcss';
+import { Modal, Portal, Text, TextInput, HelperText } from 'react-native-paper';
+import WelcomeSwiper from '../components/Swiper/WelcomeSwiper';
+import StyledButton from '../components/StyledButton';
+
+export default function OnboardingScreen() {
+  // Manage login states
+  const [loginModalVisible, setloginModalVisibility] = useState(false);
+  const [loginUsername, setLoginUsername] = useState('');
+  const [loginPassword, setLoginPassword] = useState('');
+  const showLoginModal = () => setloginModalVisibility(true);
+  const hideLoginModal = () => setloginModalVisibility(false);
+
+  // Manage signup states
+  const [signUpModalVisible, setSignUpModalVisibility] = useState(false);
+  const [signUpUsername, setSignUpUsername] = useState('');
+  const [signUpPassword, setSignUpPassword] = useState('');
+  const [signUpEmail, setSignUpEmail] = useState('');
+  const showSignUpModal = () => setSignUpModalVisibility(true);
+  const hideSignUpModal = () => setSignUpModalVisibility(false);
+  const handleCreateAnAccount = () => {
+    hideLoginModal();
+    showSignUpModal();
+  };
+  const hasErrors = (email) => {
+    return !email.includes('@');
+  };
+
+  return (
+    <View style={[t.flex1, t.justifyCenter, t.itemsCenter, t.bgGreen600]}>
+      {/* Onboarding swiper slider */}
+      <WelcomeSwiper />
+      {/* Login Modal */}
+      <Portal>
+        <Modal
+          contentContainerStyle={[t.bgWhite, t.p8]}
+          visible={loginModalVisible}
+          onDismiss={hideLoginModal}
+        >
+          <TextInput
+            label="Username"
+            value={loginUsername}
+            onChangeText={(text) => setLoginUsername(text)}
+          />
+          <TextInput
+            label="Password"
+            value={loginPassword}
+            onChangeText={(text) => setLoginPassword(text)}
+          />
```

Comment:
should have the password hidden.
Check out: https://stackoverflow.com/questions/29337444/how-do-you-style-a-textinput-in-react-native-for-password-input

Reply:
https://reactnative.dev/docs/textinput